dynamic_sampling_context.go

```diff
@@ -36,10 +37,57 @@ func NewDynamicSamplingContext(header []byte) (DynamicSamplingContext, error) {
 	}, nil
 }
 
+func DynamicSamplingContextFromTransaction(span *Span) (DynamicSamplingContext, error) {
```

Comment:
m: Should this return an error?

Reply:
https://github.com/getsentry/sentry-go/pull/492/commits/b182fc54e4fa80621648d2a6b006b359593b135b